qchem/pennylane_qchem/qchem/structure.py

```diff
@@ -414,8 +416,10 @@ def _terms_to_qubit_operator(coeffs, ops):
     for coeff, op in zip(coeffs, ops):
 
         # wire ids
-        wires = op.wires.tolist()  # Can we use subsystems here? Otherwise the Qchem library relies on users
-                                   # and devices assuming consecutive indices as wires
+        wires = (
+            op.wires.tolist()
+        )  # Can we use subsystems here? Otherwise the Qchem library relies on users
+        # and devices assuming consecutive indices as wires
```

Comment:
Yes. This might be due to a comment added by Maria or Zy regarding wires refactoring in this function that came into after I merged in the master branch.